### PR TITLE
Added security protection of the API

### LINKED ISSUE #11: Access security
If I don't miss something, this bundle allows everyone to request the API URLs. At this moment, it means one can retrieve all tree information, even from nodes that shouldn't be showed to other people.

Besides that, when we implement more things than just GET (#6), even more things will be provided to the hacker.

### PR DESCRIPTION
**Pull request by wouterj**
/fixes #11

### CODE REVIEW

File: Tests/Resources/TestBundle/Security/ResourceVoter.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+class ResourceVoter extends Voter
+{
+    protected function supports($attribute, $subject)
+    {
+        return in_array($attribute, ['CMF_RESOURCE_READ', 'CMF_RESOURCE_WRITE']);
+    }
+
+    protected function voteOnAttribute($attribute, $subject, TokenInterface $token)
+    {
+        if ('/cms/public' !== substr($subject['path'], 0, 11)) {
```
Comment (dantleech) [inline Tests/Resources/TestBundle/Security/ResourceVoter.php]: What does this path correspond to? should it be configured?

File: Tests/Resources/TestBundle/Security/ResourceVoter.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            return false;
+        }
+
+        if ('CMF_RESOURCE_WRITE' === $attribute) {
```
Comment (dantleech) [inline Tests/Resources/TestBundle/Security/ResourceVoter.php]: Should this be a constant?

File: Tests/Resources/TestBundle/Security/ResourceVoter.php (hunk 1 above)
Comment (wouterj) [reply Tests/Resources/TestBundle/Security/ResourceVoter.php]: The requested resource path to the controller. I have no idea what you mean with "configured". 

*Please be aware that you're now looking at test fixture code.*

Comment (dantleech) [reply Tests/Resources/TestBundle/Security/ResourceVoter.php]: > Please be aware that you're now looking at test fixture code.

Yep, ignore me.

**Pull request merged**